sdk/eventgrid/tests.yml

```diff
@@ -3,6 +3,9 @@ trigger: none
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
+    MaxParallel: 4
```

Comment:
Is there a reason for explicitly setting this to 4?

Reply:
this is copy from servicebus setting, remove this as this might not mandatory